src/components/LayoutComponent/LayoutComponent.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Footer from '../Footer/Footer';
+import Header from '../Header/Header';
+
+type Props = {
+  children: JSX.Element,
+};
```

Comment:
remove to `LayoutComponent.types.tsx`

Reply:
I changed from mui/joy to mui library and concept. I have added the changes for approval.